### PR TITLE
Fix ISOMIP+ docs

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] User's Guide has been updated
* [x] Developer's Guide has been updated
* [x] API documentation in the Developer's Guide (`api.rst`) has any new or modified class, method and/or functions listed
* [x] Documentation has been [built locally](https://mpas-dev.github.io/compass/latest/developers_guide/building_docs.html) and changes look as expected
* [x] Document (in a comment titled `Testing` in this PR) any testing that was used to verify the changes

### CODE REVIEW

File: docs/developers_guide/ocean/test_groups/isomip_plus.rst [hunk 1]
```diff
@@ -33,11 +33,13 @@ rising indefinitely due to the input of freshwater from ice-shelf melting.
...
-thickness, then apply smoothing to the topography data.  Optionally, the
-ice draft can be scaled by a factor as a simple way to explore changing
-ice-shelf topography.
+The function :py:func:`compass.ocean.tests.isomip_plus.geom.define_thin_film_mask_step1()`
+interpolates the ocean mask from the original BISICLES grid to the MPAS mesh
+in cases with a thin film. This is handled separately from other fields because
+the ocean mask is needed to cull land cells from the MPAS mesh before
+interpolating the remaining fields and because the thin-film region needs to
+be preserved.
+
```
Comment (xylar) [inline docs/developers_guide/ocean/test_groups/isomip_plus.rst:42]: @cbegeman, I'm not sure this is accurate.  Can you verify?

File: compass/ocean/tests/isomip_plus/geom.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ def define_thin_film_mask_step1(ds_mesh, ds_geom):
 
     ds_geom : xarray.Dataset
         Ice-sheet topography produced by
-        :py:func:`compass.ocean.tests.isomip_plus.geom.process_input_geometry()`
+        :py:class:`compass.ocean.tests.isomip_plus.process_geom.ProcessGeom`
```
Comment (xylar) [inline compass/ocean/tests/isomip_plus/geom.py:20]: @cbegeman, above here, I think the docstirng just got copied from `interpolate_ocean_mask()`.  Maybe that's fine but can you double check?

**Comment by xylar** [comment]
## Testing

I built the documentation locally and complaints about anm incorrect function name in the API documentation went away.

File: compass/ocean/tests/isomip_plus/geom.py (hunk 2 above)
Comment (cbegeman) [reply compass/ocean/tests/isomip_plus/geom.py:20]: This change looks right. I think the function change happened with your reorganization cea496c4fda1a215fa27e0bb238e425b78c1c461.

File: docs/developers_guide/ocean/test_groups/isomip_plus.rst [hunk 3]
```diff
@@ -33,11 +33,13 @@ rising indefinitely due to the input of freshwater from ice-shelf melting.
...
-is used to apply a simple calving scheme based on a threshold in ice-shelf
-thickness, then apply smoothing to the topography data.  Optionally, the
-ice draft can be scaled by a factor as a simple way to explore changing
-ice-shelf topography.
+The function :py:func:`compass.ocean.tests.isomip_plus.geom.define_thin_film_mask_step1()`
+interpolates the ocean mask from the original BISICLES grid to the MPAS mesh
+in cases with a thin film. This is handled separately from other fields because
+the ocean mask is needed to cull land cells from the MPAS mesh before
+interpolating the remaining fields and because the thin-film region needs to
+be preserved.
```
Comment (cbegeman) [inline docs/developers_guide/ocean/test_groups/isomip_plus.rst:41]: The description does need to be modified in the code as well.

```suggestion
The function :py:func:`compass.ocean.tests.isomip_plus.geom.define_thin_film_mask_step1()`
defines an MPAS mesh mask for the ocean domain including cells over the full x- and y-range
in cases with a thin film. Thus, all land cells are included in the ocean's thin-film region.
```

**Review by cbegeman — APPROVED** [review]
@xylar Thanks for doing this fixup and for catching the out of date description of the thin film mask step! Let me know if you'd like me to try building the docs on this branch.

File: compass/ocean/tests/isomip_plus/geom.py (hunk 2 above)
Comment (xylar) [reply compass/ocean/tests/isomip_plus/geom.py:20]: Yeah, there isn't a way to comment on lines that weren't changed.  This is very useful!

**Comment by xylar** [comment]
@cbegeman, thanks very much for your review and for the input.

**Pull request merged**